lib/modules.js

```diff
@@ -115,19 +115,34 @@ async function getLatestDistTag (repo) {
   return body.tag_name
 }
 
-const lastModuleDistTag = new Map()
+async function getLastSeenModuleDistTag ({ module, latestModuleVersions }) {
```

Comment:
I find it a bit confusing that `latestModuleVersions` is a directory path. When I see this name, I expect a collection of version numbers.

How about changing this name to something like `latestVersionsDir` or `moduleVersionsDir`? `latestModuleVersionsDir` is an option too, although a bit long.

Reply:
I found this confusing as well, thanks will change!